bin/init.sh

```diff
@@ -18,31 +18,25 @@ pushd $(dirname "$0")/.. > /dev/null
 
 slug="$product_name-$component_name"
 
-declare -a files_with_port=(.env Dockerfile README.md src/main/resources/application.yaml charts/rpe-spring-boot-template/values.yaml)
-declare -a files_with_slug=(build.gradle docker-compose.yml Dockerfile README.md ./infrastructure/main.tf ./src/main/java/uk/gov/hmcts/reform/demo/controllers/RootController.java charts/rpe-spring-boot-template/Chart.yaml)
+declare -a files_with_port=(.env Dockerfile README.md src/main/resources/application.yaml charts/civil-sdt-commissioning/values.yaml)
+declare -a files_with_slug=(build.gradle docker-compose.yml Dockerfile README.md ./infrastructure/main.tf ./src/main/java/uk/gov/hmcts/reform/demo/controllers/RootController.java charts/civil-sdt-commissioning/Chart.yaml)
 
 # Replace in CNP file
 for i in "Jenkinsfile_template"
 do
-  perl -i -pe "s/rpe/$product_name/g" ${i}
+  perl -i -pe "s/civil/$product_name/g" ${i}
   perl -i -pe "s/demo/$component_name/g" ${i}
 done
 
 # Replace image repo
-for i in "charts/rpe-spring-boot-template/values.yaml"
+for i in "charts/civil-sdt-commissioning/values.yaml"
 do
-  perl -i -pe "s/rpe/$product_name/g" ${i}
-  perl -i -pe "s/spring-boot-template/$component_name/g" ${i}
+  perl -i -pe "s/civil/$product_name/g" ${i}
+  perl -i -pe "s/sdt-commissioning/$component_name/g" ${i}
```

Comment:
sdt-commissioning -->  civil-sdt-commissioning?

Reply:
Wasn't sure about this but done